app/helpers/smart_proxies_helper.rb

```diff
@@ -3,12 +3,6 @@ module SmartProxiesHelper
 
   def proxy_actions(proxy, authorizer)
     [ display_link_if_authorized(_("Edit"), hash_for_edit_smart_proxy_path(:id => proxy), :class => 'edit_two_pane') ] +
-    [if proxy.has_feature?('Puppet CA')
```

Comment:
this was useful for direct navigation, any chance to keep this kind of actions (one click vs multiple)